apps/accounts/permissions.py

```diff
@@ -0,0 +1,27 @@
+from dataclasses import dataclass
+
+
+"""
+Definitions for object-level permissions for the `accounts` app.
+"""
+
+
+@dataclass
+class Permission:
```

Comment:
I am a bit confused about the difference between this and [the Django Permission model](https://docs.djangoproject.com/en/4.1/ref/contrib/auth/#permission-model), and why the dataclass appears to be being used instead of the normal Permission model.

Would you add a comment or explain what's going on here? I wouldn't feel that confident explaining this part to someone else right now.


We have this dataclass to